### DISCUSSION TITLE
Installation errors

### DISCUSSION THREAD

**Discussion by clff** [vueform/vueform#25] [Questions] [answered]
when installing vueform on in my laravel/inertia project, i'm getting this console error which crashed the entire app.

Uncaught SyntaxError: The requested module '/node_modules/.pnpm/wnumb@1.2.0/node_modules/wnumb/wNumb.js?v=c124c8fb' does not provide an export named 'default' (at useTooltip.js?v=c124c8fb:2:8)

I'm on a mac, using vuejs and tailwind, installing via pnpm.

Any help would be appreciated.

**Comment by adamberecz** [comment, marked as answer]
It seems like `vite.config.js` plugin was missing from our installation guide. It's now patched. You need to add the `vueform` plugin:

```js
// vite.config.js

import { defineConfig } from 'vite'
import laravel from 'laravel-vite-plugin';
import vueform from '@vueform/vueform/vite'

export default defineConfig({
  plugins: [
    laravel({
      input: ['resources/css/app.css', 'resources/js/app.js'],
      refresh: true,
    }),
    vueform(),
  ]
})
```
